### PR TITLE
Fix pipeline issues

### PR DESCRIPTION
**Pull request by DDavidPrime**
Addresses pipeline issues regarding deprecated theme parameters.

### CODE REVIEW

**Review by StanevPrime — CHANGES_REQUESTED** [review]
The pipelines are still failing

**Comment by DDavidPrime** [comment]
Since the only remaining job is the one failing due to some goldens, this will be handled in a separate PR.
We need the to update the goldens, but in order to do so, we need to re-generate them.
I've added a workflow which generates goldens, but this option will be available only after it is merged to the default branch (source https://stackoverflow.com/a/71423764).

File: packages/rx_bloc_generator/lib/src/builders/state_method.dart [hunk 1]
```diff
@@ -11,6 +11,7 @@ class _StateMethod implements _BuilderContract {
         (b) => b
           ..docs.addAll(['']) // A new line
           ..returns = refer(
+            // ignore: deprecated_member_use
```
Comment (StanevPrime) [inline packages/rx_bloc_generator/lib/src/builders/state_method.dart:14]: Why we ignore the warning instead of fixing it ?

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: packages/rx_bloc_generator/lib/src/builders/state_method.dart (hunk 1 above)
Comment (DDavidPrime) [reply packages/rx_bloc_generator/lib/src/builders/state_method.dart:14]: Well, this issue is only present in the pipeline and not locally (using the same flutter version!). The analysis warnings present `withNullability` option in `getDisplayString(withNullability: true)` as deprecated even though the official documentation mentions nothing and treats the method as valid:

Screenshot 2024-05-31 at 13 16 36

This ignores the pipeline warnings without any effect on the `rx_bloc_cli` package.

Comment (StanevPrime) [reply packages/rx_bloc_generator/lib/src/builders/state_method.dart:14]: Please check the flutter and packages versions are the same as on your local env

Comment (DDavidPrime) [reply packages/rx_bloc_generator/lib/src/builders/state_method.dart:14]: Yep, just double checked but it seems that the flutter version, dart version as well as the packages are all the same (even when fetched using `flutter pub get` or `flutter pub upgrade`).

**Review by StanevPrime — APPROVED** [review]

**Pull request merged**